src/slic3r/GUI/GUI_ObjectLayers.cpp

```diff
@@ -27,7 +28,11 @@ ObjectLayers::ObjectLayers(wxWindow* parent) :
     m_grid_sizer->AddGrowableCol(1);
     m_grid_sizer->AddGrowableCol(3);
 
-    m_og->activate();
+    //m_og->activate();
+    LabeledStaticBox* stb = new LabeledStaticBox(parent);
```

Comment:
Not sure if this is needed, since this is exactly what `m_og->activate()` does except the `SetCornerRadius`.